helm/prometheus-operator-app/templates/crd-install/crd-rbac.yaml

```diff
@@ -26,6 +26,12 @@ rules:
   verbs:
   - list
   - delete
+- apiGroups:
+  - application.giantswarm.io
+  resources:
+  - charts
+  verbs:
+  - get
```

Comment:
is `get` enough here? Does the `kubectl patch` work?

Reply:
We `get` the chart and `patch` the crd. So it works yes.